src/components/ActNowPage/index.js

```diff
@@ -142,6 +145,18 @@ function ActNow({ actNow, footer, title, navigation, description, ...props }) {
                 </Grid>
               </Grid>
             }
+            content={
+              <>
+                <Grid container spacing={5}>
```

Comment:
```suggestion
                <Grid container>
```
Lets not use spacing because we can't control it for various screen sizes.